pages/en/lb4/Context.md

```diff
@@ -48,6 +51,36 @@ In this case, you are using the `.controller` helper method to register a new
 controller. The important point to note is `MyController` is actually registered
 into the Application Context (`app` is a Context).
 
+## Server-level context
+
```

Comment:
Also add "Server-level context:" as a lead-in paragraph... It makes sentences with the two bullet-points (the heading is separate).